storage-api-interface.go

```diff
@@ -16,10 +16,17 @@
 
 package main
 
-import "io"
+import (
+	"io"
+
+	"github.com/minio/minio/pkg/disk"
+)
 
 // StorageAPI interface.
 type StorageAPI interface {
+	// Storage disk operations.
+	GetDiskInfo() (disk.Info, error)
```

Comment:
> Do we keep such management APIs in storage-api-interface or a separate management-api-interface?

GetDiskInfo this is the right place for now, since we don't have management APIs this will do.